### PR TITLE
HC-1654: fixes deployment

### PR DESCRIPTION
**Pull request by Glumli**

### CODE REVIEW

**Comment by Glumli** [comment]
@theintz @Harsimran1 @pussinboots PTAL
we will have to update the environment, but the dev credentials from 1PW failed

File: .travis.yml [hunk 1]
```diff
@@ -11,66 +11,62 @@ before_install:
...
-        - npm run build:${ENV}
+        - npm run build
+        - sha256sum dest/healthcloud_sdk.js > dest/healthcloud_sdk.js.sha256
       deploy:
         provider: script
         script: scripts/publish_sdk.sh ${ENV}
         skip_cleanup: true
         on:
-          branch: master
+          branch: js_th/improves-deployment
```
Comment (Harsimran1) [inline .travis.yml]: Make sure to change this to master

File: .travis.yml [hunk 2]
```diff
@@ -11,66 +11,62 @@ before_install:
...
     - stage: build production
       node_js: "7"
       env:
-      - ENV=sandbox # change to PRODUCTION as soon as endpoint is available
+        - ENV=dev # change to PRODUCTION as soon as endpoint is available
```
Comment (Harsimran1) [inline .travis.yml]: do we want to change this to `prod` now?

**Comment by pussinboots** [comment]
The production endpoint is available so you can change that
- ENV=dev # change to PRODUCTION as soon as endpoint is available
to
- ENV=prod # change to PRODUCTION as soon as endpoint is available.

File: .travis.yml (hunk 1 above)
Comment (Glumli) [reply .travis.yml]: Good catch

**Comment by Glumli** [comment]
@pussinboots You're totally right. Can you please make this change, as I don't have access to the azure credentials for prod?

**Pull request merged**